uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/leave-absences/staff/my-leave-calendar.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var page = require('../../../page-objects/ssp-leave-absences');
+module.exports = function (casper) {
+  page.init(casper).initPage()
+    .then(function () {
+      page.init(casper).allMonths();
```

Comment:
it's not clear what this does. All months what? Select all months? Show all months? i checked the method itself and it's not clear also from there